app/controllers/auth_controller.rb

```diff
@@ -70,8 +70,14 @@ def create_key
     
 
     @options = WebAuthn::Credential.options_for_create(
-      user: { id: user.webauthn_id, name: user.email }
-    )
+      user: { id: user.webauthn_id, name: user.email, display_name: user.name },
+      authenticator_selection: {
+        residentKey: "required",
```

Comment:
no it means that obl.ong requires discoverable credentials (or resident keys), which are the only type that obl.ong works with anyway. it's for ensuring that the yubikey makes one of those instead of non-resident-keys which doesn't work.